### PR TITLE
fixed issues of job-detail page

### PR DESCRIPTION
**Pull request by marjan-ara**
This PR is created to fix issues mention here:
https://www.notion.so/socious/U-job-listing-job-detail-page-853feb5dd2db47be915fdcb251b360ce

### CODE REVIEW

File: src/Nowruz/modules/userProfile/components/mainInfo/index.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ export const MainInfo = () => {
       {type === 'users' && user.languages && <LanguageJSX items={user.languages || []} />}
       {org.industry && renderData('Industry', 'globe-04', org.industry)}
       {size && renderData('Size', 'users-01', size)}
-      {org.website && <Website website={org.website ?? ''} />}
+      {org.website && <Website url={org.website ?? ''} />}
```
Comment (jeyem) [inline src/Nowruz/modules/userProfile/components/mainInfo/index.tsx:75]: @bastantebom this is what I meant when I'm telling don't touch old components you just fix this component on Org profile and it's been used here also and Thanks to @marjan-ara it's been fixed

**Review by jeyem — APPROVED** [review]

File: src/Nowruz/modules/userProfile/components/mainInfo/index.tsx (hunk 1 above)
Comment (bastantebom) [reply src/Nowruz/modules/userProfile/components/mainInfo/index.tsx:75]: @jeyem I told you already this is not an old component, this is new! I created this component. I know the usage of this

Comment (bastantebom) [reply src/Nowruz/modules/userProfile/components/mainInfo/index.tsx:75]: We already discussed this on other PR. @jeyem

Comment (bastantebom) [reply src/Nowruz/modules/userProfile/components/mainInfo/index.tsx:75]: Screenshot 2024-01-21 at 12 56 33 pm

**Review by bastantebom — APPROVED** [review]

**Pull request merged**